ddtrace/utils/uwsgi.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from ddtrace import _worker
+from ..internal.logger import get_logger
+
+logging.basicConfig(level=logging.DEBUG)
+
+log = get_logger(__name__)
+
+
+def check_threads_enabled():
+    """
+    This function is a helper used to confirm whether or not the traced application is 
```

Comment:
Can you break this up into multiple lines, e.g.:

```python
def my_func():
    """
    One sentence briefly describing the function.

    Longer sentence/paragraphs (reStructuredText/RST compatible) describing in more details.
    """
```

Reply:
Making a few revisions to syntax based on this:
https://devguide.python.org/documenting/#inline-markup

Reply:
Also broke up a bit.